cloudbridge/cloud/providers/gce/services.py

```diff
@@ -774,10 +773,13 @@ def list(self, network=None, region=None, limit=None, marker=None):
                                      response.get('nextPageToken'),
                                      False, data=subnets)
 
-    def create(self, network, cidr_block, name=None, zone=None):
+    def create(self, network, cidr_block, name=None, zone=None, region=None):
```

Comment:
Could you add a comment explaining how regions and zones are intended to work here.
I'm not sure if the implementation reflects what had been discussed.

I think the intended implementation is:

```
# GCE subnets are regional. The logic for determining the subnet region here is:
# - If a region is provided, use it.
# - If a zone is provided, use the region that it belongs to.
# - If neither is provided, use the default region as set in the provider.
```

Reply:
Also, `region` is not a method param in the interface (https://github.com/gvlproject/cloudbridge/blob/master/cloudbridge/cloud/interfaces/services.py#L728). Is it not possible to obtain the region from the zone object? and use the _default_ region when the zone is not provided?

Reply:
Done: fixed in #74.